.github/PULL_REQUEST_TEMPLATE/register_did_method.md

```diff
@@ -0,0 +1,47 @@
+## New DID Method Registration
```

Comment:
We need to agree to a documented process before we can change the registry to support JSON files, please review this section.